docs/setup/kubeedge_install_source.md

```diff
@@ -0,0 +1,163 @@
+# Setup from Source Code
+
+This guide would provide steps which can be utilised to install KubeEdge Cloud and Edge side. At this point, we assume that you would have installed the Pre-Requisites for Cloud and Edge.
+
+## Setup Cloud Side (KubeEdge Master Node)
+
+1. Clone KubeEdge
+
+    + Before this it might be a good idea to setup [$GOPATH ](https://github.com/golang/go/wiki/SettingGOPATH)
```

Comment:
**it might be a good idea** implies conjecture. If the user must follow the step then put it as a pre requisite. If not Just say **Setup GOPATH** and why.